layouts/blog/content.html

```diff
@@ -0,0 +1,25 @@
+<div class="td-content">
+	<h1>{{ .Title }}</h1>
+	{{ with .Params.description }}<div class="lead">{{ . | markdownify }}</div>{{ end }}
+	<div class="td-byline mb-4">
+		{{ with .Params.author }}{{ T "post_byline_by" }} <b>{{ . | markdownify }}</b> |{{ end}}
+    {{ if .Params.translator }}
+      Translated {{ with .Params.translator }}{{ T "post_byline_by" }} <b>{{ . | markdownify }}</b> |{{ end }}
```

Comment:
Please localize the hardcoded string "**Translated**" by adding it to the **[en.toml](https://github.com/kubernetes/website/blob/main/data/i18n/en/en.toml)** file and use the Hugo translate 'T' [function](https://gohugo.io/functions/lang/translate/) to retrieve its value.

Reply:
Thanks, fixed by 3444b26!